apis/pubsub/v1alpha1/topic_types.go

```diff
@@ -47,7 +47,17 @@ type TopicParameters struct {
 	// The expected format is `projects/*/locations/*/keyRings/*/cryptoKeys/*`.
```

Comment:
Quoting `status.atProvider.name` field comment which is what's returned by `CryptoKeyRRN()` function:
> // Name: Output only. The resource name for this CryptoKey in the
> // format
> // `projects/*/locations/*/keyRings/*/cryptoKeys/*`.